bootstrap.sh

```diff
@@ -116,12 +136,12 @@ if [ ! -d appscale ]; then
         # We split the commands, to ensure it fails if branch doesn't
         # exists (Precise git will not fail otherwise).
         git clone ${APPSCALE_REPO} appscale
-        (cd appscale; git checkout ${APPSCALE_BRANCH})
+        (cd appscale; git checkout ${APPSCALE_BRANCH}; git fetch --all)
```

Comment:
We probably just want to do `get fetch origin` here. `--all` is for all remotes.